app/frontend/src/shareUrl.js

```diff
@@ -1,9 +0,0 @@
-import ClipboardJS from 'clipboard'
-
-$(document).ready(function(){
-  new ClipboardJS('.copy-to-clipboard');
-
-  $('.copy-to-clipboard').click(function(event) {
```

Comment:
This is used in `app/views/hiring_staff/vacancies/_show.html.haml`

Reply:
Whoops!